include/miral/miral/decorations.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright © Canonical Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 or 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef MIRAL_DECORATIONS_H
+#define MIRAL_DECORATIONS_H
+
+
+#include <memory>
+
+namespace mir { class Server; }
+
+namespace miral
+{
+class Decorations
```

Comment:
```suggestion
/// Configures the window decoration strategy.
/// \note The strategy can only be applied to clients that are able to negotiate the decoration style with the server.
/// \remark Since MirAL 5.1
class Decorations
```

Reply:
I don't really have any more info to document on my mind right now. Any suggestions?